### DISCUSSION TITLE
How to reinstall Scoop with all applications into a new OS/PC?

### DISCUSSION THREAD

**Discussion by ZzZombo** [ScoopInstaller/Scoop#5567] [Help and Questions] [answered]
So I'm about to move into a new OS, how do I cleanly transfer Scoop alongside? Some applications need to be installed anew to be fully operational in the new environment, so I'd need to have Scoop reinstall them, but also I do not want to uninstall because it gets rid of all versions that I sometimes need plus sometimes I make on-the-fly custom versions of neglected applications alongside the original. Some kinda way to just run the (pre)-installer steps perhaps?

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/Scoop/discussions/5542#discussioncomment-6199111

If the **Scoop root directory** is default(`~\scoop`), then step 2 is unnecessary, this will restore most of the apps. as you might expect, some apps may require to reinstall.

**Comment by ZzZombo** [comment]
I need Scoop to reinstall the apps ignoring the fact they are already installed, the linked discussion/reply doesn't seem to deal with that.

**Comment by HUMORCE** [comment, marked as answer]
`scoop update <app> --force`

**Reply by MellowPhi** [reply]
> `scoop update <app> --force`

Just what I was looking for. I had to reset Windows and all scoop apps were still in the system! <3
